### ISSUE TITLE
'Update profile picture' when using Apache authorization

### ISSUE THREAD

**Issue by KamAZenok** [PHPfox-Official/phpfox-v4-issues#2620]
# Important

Please follow this template!

----------------------------------------

## What's happened?
When I use Apache authorization I cannot see photo to reposition in the "Update profile picture" window.

## Steps to reproduce:
1. Create '/etc/httpd/pass_file' file with the username and password using 'htpasswd' utility

2.  Create .htaccess file in the root directory of the site with the following content:

AuthType Basic
AuthName "Please log in"
AuthUserFile /etc/httpd/pass_file
Require valid-user

Or place this content into '/etc/httpd/conf/httpd.conf' file.

3. Restart Apache.

4. Log in to the page using username and password from the 1st step.

5. Enter the site with your account.

6. Go to user profile page and try to update user profile picture.

7. You will see image as in screenshot with no profile picture visible.

## Browsers and Devices tested
Chrome, Microsoft Edge on Windown 10, Chrome Mobile on Android

## Server information
CentOS 7, php 7.2.17, MySQL 5.5.5-10.3.14-MariaDB, Apache)

## phpFox version
phpFox 4.7.4 (build 2)

## Screenshots
![1](https://user-images.githubusercontent.com/44835622/56030126-c497cf80-5d24-11e9-893a-b210aa42a875.jpg)

**Comment by AmineBAIK1** [comment]
I had the same issue but with AWS...

**Comment by KamAZenok** [comment]
Did you fixed it?

**Comment by AmineBAIK1** [comment]
Yeah it's about S3 Auth Buckets ..Look in ur bucket ---> Authorisation --> CORS Config and copy past this :: 
<?xml version="1.0" encoding="UTF-8"?>
<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
<CORSRule>
    <AllowedOrigin>*</AllowedOrigin>
    <AllowedMethod>GET</AllowedMethod>
    <AllowedMethod>HEAD</AllowedMethod>
    <MaxAgeSeconds>3000</MaxAgeSeconds>
    <AllowedHeader>Authorization</AllowedHeader>
</CORSRule>
</CORSConfiguration>

**Comment by AmineBAIK1** [comment]
<?xml version="1.0" encoding="UTF-8"?>
<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
<CORSRule>
    <AllowedOrigin>*</AllowedOrigin>
    <AllowedMethod>GET</AllowedMethod>
    <AllowedMethod>HEAD</AllowedMethod>
    <MaxAgeSeconds>3000</MaxAgeSeconds>
    <AllowedHeader>Authorization</AllowedHeader>
</CORSRule>
</CORSConfiguration>

**Comment by AmineBAIK1** [comment]
![aws](https://user-images.githubusercontent.com/46829598/56306520-f48b0c80-6142-11e9-9ad7-88fa45d1af79.png)

**Comment by KamAZenok** [comment]
Thank you for your reply.
I don't use Amazon.
Does anyone faced this issue with only Apache authorization???

**Comment by harrison05** [comment]
You should check the Ajax response when inspecting your browser (on Network tab) after upload avatar, this might have an issue with Apache server like this **"SSL received a record that exceeded the maximum permissible length."**

**Comment by hamadax** [comment]
> 
> 
> You should check the Ajax response when inspecting your browser (on Network tab) after upload avatar, this might have an issue with Apache server like this **"SSL received a record that exceeded the maximum permissible length."**

I see "Failed to connect to www.domain.com port 443: Connection refused"

How is that possible when the port is open in the firewall and I can do a port scan on it.
